### DISCUSSION TITLE
Calling packages in hosted geoprocessing tools

### DISCUSSION THREAD

**Discussion by johnpfay** [R-ArcGIS/arcgisbinding#89] [General]
Has anyone been able to publish a geoprocessing tool using the R-ArcGIS bridge and the sf package? Or any package? I've been unable to do so and am trying to figure out if it's an issue with the software or my setup or implementation. Three ESRI support technicians were unable to help.

The attached scripts runs fine locally as a script tool in ArcGIS Pro (v.3.2). It also runs fine as a geoprocessing tool - but only if I comment out line 60 which invokes the sf package. When left uncommented, I get the error "This function requires the sf package" (but only when run as a remote geoprocessing tool). I've installed the same version of R (4.3.3) and the sf package on the server machine using the same account that hosts the service (arcgis), and I've confirmed it's installed correctly by running library(sf) on the server machine.
[SelectCounty_sf.zip](https://github.com/R-ArcGIS/r-bridge/files/14640365/SelectCounty_sf.zip)

I'm at a critical juncture where I need to see if ArcGIS Enterprise can support proposed research or whether we need to explore non ESRI options, so any advice is sincerely appreciated!

**Comment by JosiahParry** [comment]
Hi John, sorry you're having troubl! Can you please email us at r_bridge@esri.com? This isn't something we'll be able to handle over a GitHub issue/discussion.
